### PR TITLE
Features/sma 115 package json

### PR DESCRIPTION
**Pull request by Aboudjem**
removed unused packages
removed package-lock.json
removed `git add .` from husky hook

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@biconomy/account-contracts",
+  "name": "@bcnmy/scw-contracts",
```
Comment (tomarsachin2271) [inline package.json:2]: Don't think we should change it right now.. need some inputs from @livingrockrises here.

**Review by tomarsachin2271 — COMMENTED** [review]
Left a comment. Other changes looks good.

File: package.json (hunk 1 above)
Comment (livingrockrises) [reply package.json:2]: we don't have any organisation on npm named 'bcnmy'. If anything we can rename on github from bcnmy -> biconomy. but on npm imo these should start with @biconomy... only

Comment (Aboudjem) [reply package.json:2]: I named it like the repo but you're right. Should we use the same name everywhere to have better consistency? How about adding `account-abstraction` to the name? Like `biconomy-account-abstraction`. What do you think?

Comment (tomarsachin2271) [reply package.json:2]: For better discoverability we can use
@biconomy/account-abstraction

Comment (livingrockrises) [reply package.json:2]: no. we are talking about scw contracts package name or organisation name?

Comment (livingrockrises) [reply package.json:2]: Adam suggested org name and @tomarsachin2271 you suggestion is mix up for a package name for account contracts. @biconomy/smart-account or @biconomy/aa-smart-account will also have good visibility.

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npm run lint:fix && git add .
+npm run lint:fix
```
Comment (livingrockrises) [inline .husky/pre-commit:4]: why remove git add?

File: package.json [hunk 3]
```diff
@@ -73,39 +82,11 @@
     "typescript": "^4.7.3",
```
Comment (livingrockrises) [inline package.json:82]: can we also update typechain versions and prepack/build scripts

**Review by livingrockrises — COMMENTED** [review]
reviewed

File: .husky/pre-commit (hunk 2 above)
Comment (Aboudjem) [reply .husky/pre-commit:4]: Removing `git add .` helps us to choose the specific files we want for commit and it stops us from adding files we don't want by mistake, automatically adding all changed files to the commit can be somehow  dangerous as you might unintentionally commit files that you didn't intend to

File: package.json (hunk 1 above)
Comment (Aboudjem) [reply package.json:2]: I will keep it unchanged for now, we'll discuss it later

**Pull request merged**